libfwupdplugin/fu-device.c

```diff
@@ -237,6 +237,8 @@ fu_device_internal_flag_to_string(FuDeviceInternalFlags flag)
 		return "md-set-vendor";
 	if (flag == FU_DEVICE_INTERNAL_FLAG_NO_LID_CLOSED)
 		return "no-lid-closed";
+	if (flag == FU_DEVICE_INTERNAL_FLAG_DISABLED)
+		return "disabled";
```

Comment:
@superm1 I wonder if `FU_DEVICE_INTERNAL_FLAG_IGNORE` might be more appropriate here?

Reply:
Ya, lets do FU_DEVICE_INTERNAL_FLAG_IGNORE -- it's saying that the device _should be_ ignored, not that it's _in_ a disabled state.